### PR TITLE
CLPool staked fees update fix

### PR DESCRIPTION
**Pull request by FonsSicca**
Implements:
- Small fixed that updated `diff` variable to return incremental values instead of totals

## Summary by CodeRabbit

* **Refactor**
  * Fee reporting now emits incremental diffs (per-event increments) instead of cumulative totals for pools and user fee metrics.
  * Aggregation and per-user stats updates accept diff-style inputs and expose new activity timestamps for accurate event sequencing.

* **Tests**
  * Test suite updated to validate incremental behavior, accumulation via applied diffs, and edge cases (missing tokens, whitelisting, zero/negative amounts).

<sub>✏️ Tip: You can customize this high-level summary in your review settings.</sub>

### CODE REVIEW

File: src/EventHandlers/CLPool/CLPoolCollectFeesLogic.ts [hunk 1]
```diff
@@ -6,19 +6,14 @@ import type {
...
-
 export interface CLPoolCollectFeesResult {
-  liquidityPoolDiff: StakedFeesTotals;
+  liquidityPoolDiff: {
+    totalStakedFeesCollected0: bigint;
+    totalStakedFeesCollected1: bigint;
+    totalStakedFeesCollectedUSD: bigint;
+    totalFeesUSDWhitelisted: bigint;
+    lastUpdatedTimestamp: Date;
+  };
```
[the remark this answers is not in the document]
Comment (WooSungD) [reply src/EventHandlers/CLPool/CLPoolCollectFeesLogic.ts]: I think thats a fine trade-off and actually it being different could be better for differentiating which values are cumulative vs incremental
